src/Tokenizer/Analyzer/NamespaceUsesAnalyzer.php

```diff
@@ -57,45 +57,125 @@ public function getDeclarationsInNamespace(Tokens $tokens, NamespaceAnalysis $na
+
+                while ($groupIndex < $groupEnd) {
+                    $chunkStart = $tokens->getNextMeaningfulToken($groupIndex);
+
+                    // Finish parsing on trailing comma (no more chunks there)
+                    if ($tokens[$chunkStart]->isGivenKind(CT::T_GROUP_IMPORT_BRACE_CLOSE)) {
+                        break;
+                    }
+
+                    $groupQualifiedName = $this->getNearestQualifiedName($tokens, $chunkStart);
+                    $imports[] = new NamespaceUseAnalysis(
+                        $type,
+                        $qualifiedName['fullName'].$groupQualifiedName['fullName'],
+                        $groupQualifiedName['shortName'],
+                        $groupQualifiedName['aliased'],
+                        true,
+                        $startIndex,
+                        $endIndex,
+                        $chunkStart,
+                        $tokens->getPrevMeaningfulToken($groupQualifiedName['afterIndex'])
+                    );
+
+                    $groupIndex = $groupQualifiedName['afterIndex'];
+                }
 
-        $type = NamespaceUseAnalysis::TYPE_CLASS;
-        for ($i = $startIndex; $i <= $endIndex; ++$i) {
-            $token = $tokens[$i];
-            if ($token->equals(',') || $token->isGivenKind(CT::T_GROUP_IMPORT_BRACE_CLOSE)) {
-                // do not touch group use declarations until the logic of this is added (for example: `use some\a\{ClassD};`)
-                // ignore multiple use statements that should be split into few separate statements (for example: `use BarB, BarC as C;`)
-                return null;
-            }
+                $index = $groupIndex;
+            } elseif ($token->equalsAny([',', ';', [T_CLOSE_TAG]])) {
+                $previousToken = $tokens->getPrevMeaningfulToken($qualifiedName['afterIndex']);
+
+                if (!$tokens[$previousToken]->isGivenKind(CT::T_GROUP_IMPORT_BRACE_CLOSE)) {
+                    $imports[] = new NamespaceUseAnalysis(
+                        $type,
+                        $qualifiedName['fullName'],
+                        $qualifiedName['shortName'],
+                        $qualifiedName['aliased'],
+                        $multi,
+                        $startIndex,
+                        $endIndex,
+                        $multi ? $index : null,
+                        $multi ? $tokens->getPrevMeaningfulToken($qualifiedName['afterIndex']) : null
+                    );
+                }
 
-            if ($token->isGivenKind(CT::T_FUNCTION_IMPORT)) {
-                $type = NamespaceUseAnalysis::TYPE_FUNCTION;
-            } elseif ($token->isGivenKind(CT::T_CONST_IMPORT)) {
-                $type = NamespaceUseAnalysis::TYPE_CONSTANT;
+                $index = $qualifiedName['afterIndex'];
+                $aliased = false;
+                $fullName = $shortName = '';
```

Comment:
I think these variables are unused leftovers of the refactoring to create `getNearestQualifiedName()`.